src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -10,6 +10,7 @@ const CpsAssetPageMain = ({ pageData }) => {
   const title = path(['promo', 'headlines', 'headline'], pageData);
   const summary = path(['promo', 'summary'], pageData);
   const metadata = path(['metadata'], pageData);
+  // const blocks = path(['content', 'model', 'blocks'], pageData);
```

Comment:
comment needs removed

Reply:
I left this on purpose as it'll be instantly used in follow up PRs